### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by TungstenX** [hypeJunction/Elgg-ui_tabs#1]
I've installed hypeUI (via elgg website) and this plug-in (via github) but now I have two nav bars.  I want to keep this plugin's nav bar so I commented out the $filter lines in `mod/hypeUI/views/default/page/layouts/content.php` and `mod/hypeUI/views/default/page/layouts/default.php`

Now the question is: Is there a better way to do this? (Deactivate either one of the sub nav bars?)

**Comment by hypeJunction** [comment]
That sound like the right solution. Can't remember anymore what this plugin does

**Comment by hypeJunction** [comment]
Right, this plugins functionality is available in Elgg core already, so you don't really need it

**Comment by TungstenX** [comment]
it is used by hypeInvite
